tests/test_modeling_flaubert.py

```diff
@@ -399,3 +399,19 @@ def test_model_from_pretrained(self):
         for model_name in FLAUBERT_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
             model = FlaubertModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
+
+
+@require_torch
+class FlaubertModelIntegrationTest(unittest.TestCase):
+    @slow
+    def test_inference_no_head_absolute_embedding(self):
+        model = FlaubertModel.from_pretrained("xlm-mlm-en-2048")
+        input_ids = torch.tensor([[0, 345, 232, 328, 740, 140, 1695, 69, 6078, 1588, 2]])
+        output = model(input_ids)[0]
+        expected_shape = torch.Size((1, 11, 2048))
+        self.assertEqual(output.shape, expected_shape)
```

Comment:
I think `output` is a `torch.tensor` so `output.size()`

Reply:
The `.shape` attribute exists on `torch.tensor` and is an alias of `.size()`